test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -776,6 +776,39 @@ def forward(self, x):
             model, inputs, f, dynamic_axes={"x": [0, 1]}, input_names=["x"]
         )
 
+    def test_col2im(self):
+        # This test can be moved to test/onnx/test_pytorch_onnx_onnxruntime.py when ORT implement ::Col2Im
+
+        # Random batched RGB 32x32 image-shaped input tensor of batch size 64
+        original_image_inputs = torch.randn((64, 3, 32, 32))
+        output_size = tuple(original_image_inputs.shape[2:])
+        kernel_size = (1, 2)
+        dilation = 3
+        padding = 2
+        stride = 1
+        model_im2col = torch.nn.Unfold(
```

Comment:
Curious about if we will need a `im2col` Op after this? Or we plan to use not col2im?

Reply:
[We have already,](https://github.com/pytorch/pytorch/blob/master/torch/onnx/symbolic_opset11.py#L1111) we have it already. im2col cannot be used to express col2im and vice-versa, though